index.bs

```diff
@@ -2,7 +2,7 @@
 Title: Constructable Stylesheet Objects
 Shortname: construct-stylesheets
 Level: 1
-Status: DREAM
+Status: CG-DRAFT
```

Comment:
Stuff in the WICG should also set a [Group](https://tabatkins.github.io/bikeshed/#metadata-group).

```suggestion
Group: WICG
Status: CG-DRAFT
```